### PR TITLE
GreasedLine - setPoints update

### PR DESCRIPTION
**Pull request by RolandCsibrei**
https://forum.babylonjs.com/t/low-level-havok-debug-lines-visualization/53484/45?u=roland

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15674/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15674/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15674/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15674/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15674/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15674/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15674/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15674/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15674/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15674/merge/testResults/webgl2playwright/index.html

**Review by Popov72 — APPROVED** [review]

File: packages/dev/core/src/Meshes/GreasedLine/greasedLineBaseMesh.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { DeepCopier } from "../../Misc/deepCopier";
 import { GreasedLineSimpleMaterial } from "../../Materials/GreasedLine/greasedLineSimpleMaterial";
 import type { AbstractEngine } from "../../Engines/abstractEngine";
 import type { FloatArray, IndicesArray } from "../../types";
+import { GreasedLineTools } from "core/Misc";
```
Comment (RaananW) [inline packages/dev/core/src/Meshes/GreasedLine/greasedLineBaseMesh.ts:12]: should be imported from the correct file directly and not the directory

**Pull request merged**